docs/content/schemas.md

```diff
@@ -108,6 +113,46 @@ Consider the following input document:
 
   This indicates the error to the Rego developer right away, without having the need to observe the results of runs on actual data, thereby improving productivity.
 
+## Using annotations on Rules to further enhance the Rego type checker
+
+A rule can be annotated with a comment of the form:
+
+#@rulesSchema=`<expression>:<path-to-schema>,...,<expression>:<path-to-schema>`
```

Comment:
The backticks aren't correct here, I think

Reply:
Previews ok to me.